Callback-Cats/nutshell
# Sc render events

# Description

EVENTS:

completed render to dom functionality 
invoked functions on main
rearranged events containers on index

Fixes # (issue)
render to dom functionality completed

## Type of change

Please delete options that are not relevant.




# Testing Instructions

git fetch --all
git checkout sc-renderEvents
pleaser check that events form and events from data appear on the dom

# Checklist:
